apollo-router/src/services/layers/apq.rs

```diff
@@ -3,22 +3,22 @@
 //!  For more information on APQ see:
 //!  <https://www.apollographql.com/docs/apollo-server/performance/apq/>
 
-use std::ops::ControlFlow;
-
+use futures::future::BoxFuture;
 use futures::stream::BoxStream;
-use moka::sync::Cache;
```

Comment:
Since we are removing `moka` here, does this mean we should be able to remove `moka` from other parts of the code base?

Reply:
yes, but we should have a larger discussion about what requirements we have on cache, and then see if we keep moka or go for something else